src/main/java/com/ericsson/ei/subscription/InformSubscriber.java

```diff
@@ -377,10 +411,12 @@ private NameValuePair extractKeyAndValue(String pair) {
     }
 
     /**
-     * Returns the base url from the notification meta. Base url is all but context path and parameters.
+     * Returns the base url from the notification meta.
      *
      * @param notificationMeta
+     *     A String containing a URL
      * @return
+     *     The base url, which is everything but context path and parameters.
```

Comment:
I don't know about this sentence. Couldn't it be expressed differently? Like maybe, "which excludes context path and parameters". The original sentence comes across a bit strange to me, like, "we do everything except 50% of all things"

Reply:
Haha, true. It didn't cross my mind. I'll fix that!